desktop/src/executor.rs

```diff
@@ -129,7 +132,7 @@ impl TaskHandle {
     );
 }
 
-pub struct WinitAsyncExecutor {
+pub struct WinitAsyncExecutor<R: PollRequester> {
```

Comment:
Should this still be called `WinitAsyncExecutor`?

Reply:
It is renamed in a later commit: https://github.com/ruffle-rs/ruffle/pull/15949/commits/3ace44006b1fba5114bba0971848a7db98fd1c08